test/client/setup-test-framework.js

```diff
@@ -56,3 +59,19 @@ global.matchMedia = jest.fn( ( query ) => ( {
 	removeEventListener: jest.fn(),
 	dispatchEvent: jest.fn(),
 } ) );
+
+// This is used by @wp-playground/client
+global.ReadableStream = ReadableStream;
+global.TransformStream = TransformStream;
+global.Worker = require( 'worker_threads' ).Worker;
+
+// This is used by @wp-playground/client
+if ( typeof global.structuredClone !== 'function' ) {
+	global.structuredClone = ( obj ) => JSON.parse( JSON.stringify( obj ) );
```

Comment:
For the protocol, this polyfill works well and we've used the same one in the Gutenberg bridge.

Reply:
Perfect!